<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>info</class>
 <widget class="QWidget" name="info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>760</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>466</width>
    <height>760</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>466</width>
    <height>760</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>421</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="texto">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="titulo">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="url">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>421</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="BotonAnterior">
      <property name="styleSheet">
       <string notr="true">border-style: solid;
border-width: 1px;
;
background-color: rgb(129, 120, 255);
background-color: rgb(255, 176, 248);
color: rgb(255, 255, 255);
background-color: rgb(7, 99, 157);
border-radius: 5px;</string>
      </property>
      <property name="text">
       <string>Anterior</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/images/images/Anterior.png</normaloff>:/images/images/Anterior.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BotonSiguiente">
      <property name="styleSheet">
       <string notr="true">border-style: solid;
border-width: 1px;
;
background-color: rgb(129, 120, 255);
background-color: rgb(255, 176, 248);
color: rgb(255, 255, 255);
background-color: rgb(7, 99, 157);
border-radius: 5px;</string>
      </property>
      <property name="text">
       <string>Siguiente</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/images/images/Siguiente.png</normaloff>:/images/images/Siguiente.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="botonVolver">
      <property name="styleSheet">
       <string notr="true">border-style: solid;
border-width: 1px;
;
background-color: rgb(0, 120, 100);
color: rgb(255, 255, 255);
border-radius: 5px;
</string>
      </property>
      <property name="text">
       <string> Volver al menu</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/images/images/Menu.png</normaloff>:/images/images/Menu.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
